ClinGen/clincoded
# Connects to #1602. Additional new users.

**Steps to test:**
1. Go to ^/users and in the search/filter text box at the upper-right, enter **stheruarumugam@gmail.com** and expect to see one entry being returned in the search result.
2. Try searching **udo.schmidt.us@gmx.de** or **angela.shogren@gmail.com**. Expect to see entries being returned for the respective search terms as well.
